packages/kbn-plugin-discovery/src/plugin_search_paths.ts

```diff
@@ -25,7 +25,7 @@ export function getPluginSearchPaths({ rootDir, oss, examples, testPlugins }: Se
     resolve(rootDir, '..', 'kibana-extra'),
     ...(testPlugins
       ? [
-          resolve(rootDir, 'test/analytics/__fixtures__/plugins'),
+          resolve(rootDir, 'test/analytics/fixtures/plugins'),
```

Comment:
I had to move the plugins so I could write Jest tests because `jest.config.js` explicitly skips `__fixtures__` dirs.